patches/woocommerce-subscriptions.0002.fix.cart-resubscribe-translation.patch

```diff
@@ -13,14 +13,12 @@ index be724d7ae..9aae0639b 100644
 +++ b/includes/class-wcs-cart-resubscribe.php
 @@ -335,7 +335,8 @@ class WCS_Cart_Resubscribe extends WCS_Cart_Renewal {
  	public function order_button_text( $place_order_text ) {
- 
+
  		if ( isset( WC()->cart ) && count( wcs_get_order_type_cart_items( 'resubscribe' ) ) === count( WC()->cart->get_cart() ) ) {
 -			$place_order_text = _x( 'Resubscribe', 'The place order button text while resubscribing to a subscription', 'woocommerce-subscriptions' );
-+			// $place_order_text = _x( 'Resubscribe', 'The place order button text while resubscribing to a subscription', 'woocommerce-subscriptions' );
 +			$place_order_text = __( 'Resubscribe', 'woocommerce-subscriptions' );
  		}
- 
+
  		return $place_order_text;
--- 
+--
```

Comment:
@bogdanarizancu ⚠️  This patch probably won't work anymore due to removal of whitespace and because the diff context line counts were not adjusted after removing the commented out line:
https://github.com/makers99/wp-cli-shared-patches/blob/f5b1da8638b095a256950fd2501f7867fb798266/patches/woocommerce-subscriptions.0002.fix.cart-resubscribe-translation.patch#L14

(If I'm not mistaken, the diff context will change to `-335,7 +335,7`, because there are the identical amount of lines afterwards.)

No need to address this now; the patch is hopefully obsolete anyway.